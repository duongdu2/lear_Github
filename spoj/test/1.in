4
4
74 93 61 57 
6
11 32 28 99 1 61 
4
57 25 40 11 
10
44 100 30 73 39 99 89 85 52 6 
