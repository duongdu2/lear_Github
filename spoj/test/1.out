228
192
122
288
